recipes/jupyterbook-latex/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterbook-latex" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8083d7aa71bb0dc31847be4edd2b15ef9d55b56a8cf1b07e49b3b627fb7e49c7
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - sphinx >=3,<4
+    - myst-nb
+    - pyyaml
+
+test:
+  requires:
+    - pytest
```

Comment:
tests are good, but I don't see them being run... if possible, it may be helpful to add `pip` and then run `pip check`.

A quick visit to https://www.marcelotrevisani.com/grayskull and a look at the diff to what you have might be of interest...

Reply:
Thanks @bollwyvl

Reply:
@bollwyvl not sure if the test ran after adding. `pip check` ran though. Should we ideally add `pytest` under the `run` key as well, so that the test suite is run?

Reply:
so to actually run the tests, I usually recommend:

```yaml
test:
  requires:
    - pip
    - pytest-cov
  commands:
    - pip check
    - pytest --cov jupyterbook_latex --cov-report term-missing:skip-covered --no-cov-on-fail --cov-fail-under <some baseline number>
```

However: the upstream `sdist` does not include the tests. These would either need to be:
- added to `MANIFEST.in` (in which case the `test` stanza would need `source_files: [tests]`
- moved in-tree, so it is discovered by `pytest --pyargs jupyterbook_latex`

Anyhow, it'd take a new upstream release, so I'd just pull/comment everything related to `pytest` at this point, and leave `pip check`.